acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -6,21 +6,10 @@
 test_name "Install Packages"
 
 step "Install puppet-agent..." do
-  opts = {
-    :puppet_collection    => 'PC1',
-    :puppet_agent_sha     => ENV['SHA'],
-    # SUITE_VERSION is necessary for Beaker to build a package download
-    # url which is built upon a `git describe` for a SHA.
-    # Beaker currently cannot find or calculate this value based on
-    # the SHA, and thus it must be passed at invocation time.
-    # The one exception is when SHA is a tag like `1.8.0` and
-    # SUITE_VERSION will be equivalent.
-    # RE-8333 may make this unnecessary in the future
-    :puppet_agent_version => ENV['SUITE_VERSION'] || ENV['SHA']
-  }
+  dev_builds_url = ENV['DEV_BUILDS_URL'] || 'http://builds.delivery.puppetlabs.net'
   agents.each do |agent|
     next if agent == master # Avoid SERVER-528
-    install_puppet_agent_dev_repo_on(agent, opts)
+    install_from_build_data_url('puppet-agent', "#{dev_builds_url}/puppet-agent/#{ENV['SHA']}/artifacts/#{ENV['SHA']}.yaml", agent)
```

Comment:
It looks like [SERVER-528](https://tickets.puppetlabs.com/browse/SERVER-528) is resolve. Given that the `install_from_build_data_url` method can take a `hosts` array, there is no longer a need to iterate over the agents list here. This block can be collapsed into a single call to `install_from_build_data_url` with the `agents` value for the `local_hosts` parameter.

Reply:
I'll add a note here so that we're aware of why we left this as is